features/admin/deactivate_supplier_user.feature

```diff
@@ -28,14 +30,31 @@ Scenario Outline: Correct users can deactivate and reactivate a supplier's contr
     | role  |
     | admin |
 
-Scenario Outline: Correct users cannot update suppliers users
+Scenario Outline: Correct users can view suppliers users
```

Comment:
Can this be "can view but not edit" or "can view but not deactivate"? i.e. something to qualify that we aren't just checking "visibility" here.

Reply:
Done, Thanks for the review! :100: